HousingFinanceInterimApi/V1/Gateways/Interface/IGoogleClientService.cs

```diff
@@ -62,6 +63,10 @@ public interface IGoogleClientService
 
         public Task DeleteFileInDrive(string fileId);
 
+        #region Clean Google Drive interface
+        public Task<IUploadProgress> UploadFileToDrive(FileInMemory fileInMemory, string targetFolderId);
+        public Task UploadFileOrThrow(FileInMemory fileInMemory, string targetFolderId);
```

Comment:
All operations marked on an interface are public. So the "public" keyword might be superfluous

Reply:
Hmm... I thought it defaults to `private` when you leave it.
But I gave it a go at removing them & building the project.
Seems like the interfaces don't complain about that.